diff --git a/node_modules/react-native/jest/setup.js b/node_modules/react-native/jest/setup.js
index 7787788..a1c046f 100644
--- a/node_modules/react-native/jest/setup.js
+++ b/node_modules/react-native/jest/setup.js
@@ -107,17 +107,22 @@ jest
       ...MockNativeMethods,
       isFocused: jest.fn(),
       clear: jest.fn(),
-      getNativeRef: jest.fn(),
+     // getNativeRef: jest.fn(),
     }),
   )
-  .mock('../Libraries/Modal/Modal', () => {
+/*   .mock('../Libraries/Modal/Modal', () => {
     const baseComponent = mockComponent('../Libraries/Modal/Modal');
     const mockModal = jest.requireActual('./mockModal');
     return mockModal(baseComponent);
   })
   .mock('../Libraries/Components/View/View', () =>
     mockComponent('../Libraries/Components/View/View', MockNativeMethods),
-  )
+  ) */
+  .mock('../Libraries/Modal/Modal', () => mockComponent('../Libraries/Modal/Modal'))
+   .mock('../Libraries/Components/View/View', () =>
+     mockComponent('../Libraries/Components/View/View', MockNativeMethods),
+   )
+   
   .mock('../Libraries/Components/AccessibilityInfo/AccessibilityInfo', () => ({
     __esModule: true,
     default: {
